MagneticField/ParametrizedEngine/interface/ParabolicParametrizedMagneticField.h

```diff
@@ -5,11 +5,16 @@
 #ifndef MagneticField_ParametrizedEngine_interface_ParabolicParametrizedMagneticField_h
 #define MagneticField_ParametrizedEngine_interface_ParabolicParametrizedMagneticField_h
 
+#include <Eigen/Core>
```

Comment:
This `#include` should not be needed, because this header does not directly depend on Eigen.
```suggestion
```